index.d.ts

```diff
@@ -120,11 +120,17 @@ declare module "hyperapp" {
     | Action<S, P>
     | readonly [action: Action<S, P>, payload: P]
 
+  // An Effecter is the function that runs an effect
+  type Effecter<S, P = any> = (
+    dispatch: Dispatch<S>,
+    payload?: P
```

Comment:
`payload?: P` doesn't pass my type tests but changing it back to `payload: P` will.